tests/performance/locust_settings.py

```diff
@@ -30,12 +29,22 @@ class LocustSettings(BaseSettings):
     LOCUST_HOST: AnyHttpUrl = Field(
         default=..., examples=["https://api.osparc-master.speag.com"]
     )
-    LOCUST_LOCUSTFILE: Path = Field(default=...)
+    LOCUST_LOCUSTFILE: Path = Field(
+        default=...,
+        description="Test file. Path should be relative to `locust_files` dir",
```

Comment:
since the path should be relative ...  perhaps you can enforce it by having a pre-validator and/or when exporting this value

Reply:
Very good point. Thanks. I will add.